content/es/02_programming-zkps-from-zero-to-hero.md

```diff
@@ -0,0 +1,752 @@
+---
+title: 'Programando ZKPs: De cero a héroe'
+date: '2025-07-23'
+tags: ['zero-knowledge']
+draft: false
+layout: PostSimple
+slug: "/es/programming-zkps-from-zero-to-hero"
```

Comment:
Slug should be without the `/es/` prefix - this is already in the folder layout

See e.g. https://raw.githubusercontent.com/zkintro/zkintro/refs/heads/main/content/zh-tw/01_friendly-introduction-to-zero-knowledge.md